src/frontend/packages/lib_common/cunningham.ts

```diff
@@ -4,6 +4,10 @@ const config = {
   themes: {
     default: {
       theme: {
+        colors: {
+          'primary-150': '#E5EEFA',
+          main: '#edf5fa',
```

Comment:
Sorry, I think I prefer `lightblue` over `main`.
I think it may be confusing to have both `primary` and `main`: when should we use one on the other ?
Also, I find `lightblue` more descriptive and less abstract.

Reply:
Yes I think you'r right, what about that, I update the `primary-100`  which was just a nuance near what we wanted:
https://github.com/openfun/marsha/pull/2479/commits/b2993b4f882545999197af6b949c8357342ab83d